src/app/services/postman-sms/postman-sms.service.ts

```diff
@@ -92,9 +95,32 @@ class PostmanSmsService {
 
         return new SmsSendError()
       },
-    ).andThen(() => {
-      return okAsync(true as const)
-    })
+    )
+      .andThen(() => {
+        return okAsync(true as const)
+      })
+      .map((result) => {
+        // Fire log sms success promise without waiting.
+        void SmsService.logSmsSend({
+          smsData,
+          smsType,
+          msgSrvcSid: 'test msgSrvcSid',
+          logType: LogType.success,
+        })
+
+        return result
+      })
+      .mapErr((error) => {
+        // Fire log sms failure promise without waiting.
+        void SmsService.logSmsSend({
+          smsData,
+          smsType,
+          msgSrvcSid: 'test msgSrvcSid',
```

Comment:
We likely don't need `msgSrvcSid` anymore. For historical context, is the old Twilio Service ID (similar to the Singpass E-Service ID) that users would need to provide for us.